spec/realtime/message.test.js

```diff
@@ -70,6 +70,50 @@ define(['ably', 'shared_helper'], function(Ably, helper) {
 		}
 	};
 
+	/*
+	 * Test that a message is not sent back to the same realtime client
+	 * when echoMessages is false (RTC1a and RTL7f)
+	 *
+	 * Test that a message is sent back to the same realtime client
+	 * when echoMessages is true (RTC1a and RTL7f)
+	 */
+	exports.publishEcho = function(test) {
+		test.expect(3);
+
+		// set up two realtimes
+		var rt1 = helper.AblyRealtime({ echoMessages: false }),
+			rt2 = helper.AblyRealtime({ echoMessages: true }),
+
+			rt1Channel = rt1.channels.get('publishecho'),
+			rt2Channel = rt2.channels.get('publishecho'),
+
+			testMsg1 = 'Hello',
+			testMsg2 = 'World!';
+
+		monitorConnection(test, rt1);
+		monitorConnection(test, rt2);
```

Comment:
BTW. I think the monitorConnection method supports `monitorConnection(test, rt1, rt2);`